config/fastly/snippets/bad_bot_detection.vcl

```diff
@@ -113,8 +113,6 @@ sub vcl_recv {
     || req.http.user-agent == "RSurf15a 41"
     || req.http.user-agent == "RSurf15a 51"
     || req.http.user-agent == "RSurf15a 81"
-    # Block Ruby bots unless they're interacting with the API
-    || (req.http.user-agent == "Ruby" && !(req.http.url ~ "^/api"))
```

Comment:
I can confirm that this appears to be blocking requests to the /api/ endpoints when user agent is Ruby